remote_attestation_sessions/src/lib.rs

```diff
@@ -16,129 +16,13 @@
 
 //! Logic for performing remote attestation in multiple sessions
 
-#![no_std]
+// The client logic requires std.
+#![cfg_attr(not(feature = "client"), no_std)]
 
-extern crate alloc;
+pub mod server;
 
-use alloc::{boxed::Box, sync::Arc, vec::Vec};
-use anyhow::bail;
-use lru::LruCache;
-use oak_remote_attestation::handshaker::{AttestationBehavior, Encryptor, ServerHandshaker};
+#[cfg(feature = "client")]
+pub mod client;
```

Comment:
Why not have two separate crates then? Especially since IIUC the server part of this crate is expected to be no_std compatible, and it is really meant for a different class of programs.

Reply:
Mirroring the other remote attestation related crates that have a client server module structure. Agreed that here two crates would be better though.